config.py

```diff
@@ -2556,6 +2598,37 @@ def get_fs_subjects_dir():
                 .resolve())
 
 
+def get_all_contrasts() -> Iterable[ArbitraryContrast]:
+    _validate_contrasts(contrasts)
+    normalized_contrasts = []
+    for contrast in contrasts:
+        if isinstance(contrast, tuple) and len(contrast) == 2:
+            normalized_contrasts.append({
+                'name': contrast[0] + "+" + contrast[1],
+                'conditions': list(contrast),
+                'weights': [1, -1],
```

Comment:
Why don't you create an `ArbitraryContrast` directly?

Reply:
Done